src/core/core.js

```diff
@@ -498,15 +497,13 @@ define(function (require) {
 
   p5.prototype.registerPreloadMethod = function(m) {
     p5.prototype._preloadMethods.push(m);
-  }.bind(this);
+  };
```

Comment:
I'm not sure why they are there. :) I was planning to do some work with these functions tomorrow actually so I can test then and confirm whether they can be removed.